src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.targets

```diff
@@ -1,23 +1,33 @@
 <Project>
 
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiNuGets)' == 'true' or '$(UseMauiCore)' == 'true' ">
-    <ProjectCapability Include="UseMauiCore" />
-    <ProjectCapability Condition=" '$(UseMaui)' == 'true' or '$(UseMauiNuGets)' == 'true' " Include="UseMaui" />
+  <!--
+      .NET MAUI telemetry and failure tracking. These are used to detect when the user specified one of the
+      various <UseMaui*> properties, but for some reason the workload was not installed correctly.
+  -->
+  <ItemGroup>
+    <ProjectCapability Condition=" '$(UseMaui)' == 'true' " Include="UseMaui" />
+    <ProjectCapability Condition=" '$(UseMauiCore)' == 'true' " Include="UseMauiCore" />
+    <ProjectCapability Condition=" '$(UseMauiNuGets)' == 'true' " Include="UseMauiNuGets" />
+    <ProjectCapability Condition=" '$(UseMauiEssentials)' == 'true' " Include="UseMauiEssentials" />
+    <ProjectCapability Condition=" '$(UseMauiAssets)' == 'true' " Include="UseMauiAssets" />
   </ItemGroup>
 
+  <!--
+      Specifying any of the <UseMaui*> properties will enable the maui workload. BundledVersions.targets enables
+      the specific parts of .NET MAUI SDK.
+  -->
   <Import
-      Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiNuGets)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true') and ($([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@'))) "
+      Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true') and ($([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@'))) "
       Project="Sdk.targets" Sdk="Microsoft.Maui.Sdk.net@MAUI_DOTNET_VERSION_MAJOR@"
   />
   <Import
       Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' or '$(UseMauiEssentials)' == 'true' or '$(UseMauiAssets)' == 'true') and ($([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_PREVIOUS_DOTNET_VERSION@'))) "
       Project="Sdk.targets" Sdk="Microsoft.Maui.Sdk.net@MAUI_PREVIOUS_DOTNET_VERSION_MAJOR@"
   />
 
-  <Import
-      Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true') and ('$(UseMauiNuGets)' != 'true') and ($([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_DOTNET_VERSION@'))) "
-      Project="Sdk.targets" Sdk="Microsoft.Maui.Resizetizer.Sdk"
-  />
+  <!--
+      Previous versions of the .NET MAUI SDK used Microsoft.Maui.Resizetizer.Sdk, but the new version uses a NuGet.
+  -->
   <Import
       Condition=" ('$(UseMaui)' == 'true' or '$(UseMauiAssets)' == 'true') and ($([MSBuild]::VersionEquals($(TargetFrameworkVersion), '@MAUI_PREVIOUS_DOTNET_VERSION@'))) "
```

Comment:
This looks like it only goes 1 version behind. Does anything support back to `net6.0` projects?

I kept `net6.0` support in the Android workload, because I didn't know yet what the official support is.

Reply:
We have tests on CI that test all this on both net6 and net7. All our tests are duplicated for the supported TFMs.

Reply:
So if I look at the workload manifest:

https://github.com/dotnet/maui/blob/ea8c7fad3a0a33b41744506757fe645a260028b4/src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

It has:

* Microsoft.Maui.Core (has the .NET 8 stuff?)
* Microsoft.Maui.Sdk.net6
* Microsoft.Maui.Sdk.net7

What are the values for `@MAUI_DOTNET_VERSION_MAJOR@` and `@MAUI_PREVIOUS_DOTNET_RELEASED_NUGET_VERSION@`? By their naming it sounds like these are 8 and 7.

Reply:
If we decide to backport this to net7 servicing, then this will involve more conditions and tests in the targets file to either install nugets for later net7 or workloads. So maybe we will have a condition that is `currentVersion = previousTFM AND currentVersion > switchVersion`